includes/html/table/alertlog.inc.php

```diff
@@ -13,15 +13,16 @@
  * @author     LibreNMS Contributors
 */
 
-$alert_severities = [
+$alert_severities = array(
     // alert_rules.status is enum('ok','warning','critical')
     'ok' => 1,
     'warning' => 2,
     'critical' => 3,
```

Comment:
moving this Array to an Enum would be appreciated
take a look on LibreNMS\Enum directory for examples